packages/adblocker/src/config.ts

```diff
@@ -61,6 +63,7 @@ export default class Config {
     loadExtendedSelectors = false,
     loadGenericCosmeticsFilters = true,
     loadNetworkFilters = true,
+    loadPreprocessors = true,
```

Comment:
I think it might be safest to turn it `off` by default for the initial release so that this change can be rolled out incrementally and controlled by the backend (to avoid any issues). And in a second release it could be turned to `true` once everything has been validated.